test/input.txt

```diff
@@ -83,9 +83,16 @@
 
   # should show all details even private for viewall
   viewall 3
-  viewall 4
+  viewall 4	
   viewall 5
 
+##########################################################
+# test sort persons command
+##########################################################
+  
+  # should sort all entries so far properly
+  sort 
+  	
```

Comment:
It might be appropriate to check that your sorting does persist by calling list. Currently as your persons are already in alphabetical order, your testing might not be able to catch problems.